app/scripts/controllers/transactions/index.js

```diff
@@ -413,10 +413,17 @@ class TransactionController extends EventEmitter {
     // sign tx
     const fromAddress = txParams.from
     const ethTx = new Transaction(txParams)
-    await this.signEthTx(ethTx, fromAddress)
+    const signedTx = await this.signEthTx(ethTx, fromAddress)
```

Comment:
We should remember to document that the plugin consumer can either return a signed hex string or the signature on the ethjs-tx object.